app.js

```diff
@@ -178,4 +178,22 @@ app.patch('/api/v1/palettes/:id', async (req, res) => {
   };
 });
 
+app.delete('/api/v1/projects/:id', async (req, res) => {
+  let { id } = req.params;
+  let foundProject = await database('projects').where('id', id).select();
+  id = parseInt(id);
+  if (!id || !foundProject.length) {
+    return res.status(404).send({
+      error: `No project with id: ${id} exists. It has either already been deleted or the id sent with request is incorrect.`
+    });
+  }
+  try {
+    await database('palettes').where('project_id', id).del();
+    await database('projects').where('id', id).del();
+    res.status(200).send({ id: id });
```

Comment:
I think we may want to use  202 status code here with a custom message indicating that the delete was successful.